ipopt-3.13.4_install

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+###############################################
+
+package_name="ipopt"
+package_version="3.13.4"
+package_description="Ipopt (Interior Point OPTimizer, pronounced eye-pea-Opt) is a software package for large-scale nonlinear optimization."
+SRC_ARCHIVE=${SRC_ARCHIVE:-git@github.com:coin-or/Ipopt.git}
```

Comment:
If you are git cloning the repo rather than downloading a release tarfile and building that, you need to make sure you are getting a specific version, so if this is rerun in a year, it gets the same code.